### DISCUSSION TITLE
openai tts issue

### DISCUSSION THREAD

**Discussion by samqin123** [open-webui/open-webui#752] [General]
# Bug Report

## Description
Using OPENAI TTS，the reply stops after reading part about 10-12seconds, might be token-limitation, i cannot tell.

**Bug Summary:**
[Provide a brief but clear summary of the bug] Using OPENAI TTS，the reply stops after reading part about 10-12seconds, might be token-limitation, i cannot tell.

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

**Expected Behavior:**
[Describe what you expected to happen.] might be workable to adjust the token limitation or change configuration. butter or streaming support will be necessary.

**Actual Behavior:**
[Describe what actually happened.]

## Environment

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I have reviewed the troubleshooting.md document.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Hi, could you provide steps to reproduce the issue?

**Reply by nijisakai** [reply]
i also met this problem. #1442
